### DISCUSSION TITLE
Facing Error with EMAILOCTOPUS Newsletter

### DISCUSSION THREAD

**Discussion by ashishlotake** [timlrx/tailwind-nextjs-starter-blog#537] [Q&A]
My *siteMetadata.js*

![Screenshot from 2022-09-02 00-26-22](https://user-images.githubusercontent.com/62111173/187992836-543097a7-8a94-4786-a63b-7b3260c8e562.png)

the *emailoctopus.js*
![Screenshot from 2022-09-02 00-27-08](https://user-images.githubusercontent.com/62111173/187992978-f9f82f78-cc94-4c14-a5d0-ee482628b14e.png)

The added environment variable in vercel 

![Screenshot from 2022-09-02 00-29-03](https://user-images.githubusercontent.com/62111173/187993061-2976caf4-2e37-498e-92da-78d0a744851a.png)

Still Newsletter is not working
error in browser

![Screenshot from 2022-09-02 00-31-15](https://user-images.githubusercontent.com/62111173/187993147-b828392e-8eab-412f-b9aa-9e05dcb2c454.png)

If **.env.example**  makes any difference, its there in my repo.

**Comment by timlrx** [comment]
Strange, it should work, I am using this on my personal blog as well though I am using `https://emailoctopus.com/api/1.5/`  as the API URL, though from a quick look at the documentation, that does not seem to make a difference for this API route. 

Notice that you have switched to a different provider, so I won't investigate further.
